frontend/hub/useHubView.tsx

```diff
@@ -22,6 +22,11 @@ export interface HubItemsResponse<T extends object> {
   };
 }
 
+export interface HubNamespaceResponse<T extends object> {
+  count: number;
+  results: T[];
+}
+
```

Comment:
you may want to replace this one with `PulpItemsResponse` from `usePulpView` - that matches the `{count, results[], next?}` format